src/block/SnowLayer.php

```diff
@@ -105,6 +105,14 @@ public function ticksRandomly() : bool{
 		return true;
 	}
 
+	public function getFallDamagePerDistance() : float{
+		return 0.0;
+	}
+
+	public function getMaxFallDamage() : float{
+		return 0.0;
+	}
+
```

Comment:
These default implementations can be added to `FallableTrait` to avoid polluting these classes.